akka-actor/src/main/scala/akka/serialization/Serialization.scala

```diff
@@ -188,8 +188,9 @@ class Serialization(val system: ExtendedActorSystem) extends Extension {
     Try {
       val serializer = try getSerializerById(serializerId) catch {
         case _: NoSuchElementException ⇒ throw new NotSerializableException(
-          s"Cannot find serializer with id [$serializerId]. The most probable reason is that the configuration entry " +
-            "akka.actor.serializers is not in synch between the two systems.")
+          s"Cannot find serializer with id [$serializerId] ${clazz.map(c ⇒ "(class [" + c.getName + "]").getOrElse()}. " +
```

Comment:
```suggestion
          s"Cannot find serializer with id [$serializerId]${clazz.map(c ⇒ " (class [" + c.getName + "])").getOrElse("")}. " +
```
I think `None.getOrElse().toString` would be `"()"` where I guess we'd want `""`?

Reply:
ah, ups should be ""